### PR TITLE
Using modules

### PR DESCRIPTION
**Pull request by BezzB**
In this project I have

Created a new directory and initialize it with git.
Use npm init -y command to create package.json file.
Created the entry point for your JavaScript code called index.js
Created index.css file for your styles.
Created the main index.html file. Link your index.js and index.css files there.
Created a directory called modules.
Re-created your Awesome books app following the rules:

Use ES6 syntax.
Divided your code into modules.
Created a separate file for each module in the modules directory.
Imported modules into the index.js file. (Tip: the more modules the better!)
Kept all funcionalities of the app without errors.
Refactored your functions to arrow functions.
Used let and const in the correct way.
Displayed the current date below the navigation bar.

For this requirement you should use a library [Luxon](https://moment.github.io/luxon/#/install?id=es6)
Download the library into your project using npm.
Import it as a module.
See the article [Get the current date and time](https://moment.github.io/luxon/#/tour?id=get-the-current-date-and-time). Your task here is to use a new tool effectively with little externally curated support. Exactly as you will do in your future job.

### CODE REVIEW

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# INVALID ❌ ❌ ❌ 

Hi @BezzB, 👋

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request ❌
 - Unfortunately, your project does not have a proper README file following the Microverse [README template](https://github.com/microverseinc/readme-template). Please add the README and customize all the sections to match this project. Please remove the Microverse logo.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

✅ Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_⚠️ Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

----
_Invalid Code Review Request does not count into the code reviews limit._

File: index.html [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+    <script src="index.js" type="module"></script>
+    <script src="https://cdn.jsdelivr.net/npm/luxon@2.3.0/build/global/luxon.min.js"></script>
+<script>
+  // Get the current date and format it using Luxon
+  const currentDate = luxon.DateTime.local().toFormat('EEEE, LLLL d, y');
+
+  // Update the current date element with the formatted date
+  const currentDateElement = document.getElementById('current-date');
+  currentDateElement.innerText = currentDate;
+</script>
```
Comment (leonardodiasb) [inline index.html]: - [ ] The purpose of this project is to start developing the SOLID mentality and modularize your project so it become more organized. The date is correctly displayed on the UI, but kindly import the module from Luxon in your `index.js` file, e.g:`import { DateTime } from "luxon";`. Please, download it using `npm` and use it as a module as the project required.

**Review by leonardodiasb — CHANGES_REQUESTED** [review]
# Required Changes ♻️ ⚠️ 🚦 

Hi @BezzB, 

Good job so far!
You still need to work on some issues to go to the next project, but you are almost there!

mario_ladders

## Good points 👍 

 - No linter errors. ✔️
 - Descriptive PR. ✔️ 

## Aspects to improve ♻️

- [x] Check the comments under the review.
- [ ] There is an empty file in your modules directory. Kindly delete unused files, since they don't add functionality to your project, they are not necessary and can make your project look messy.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think the code review was unfair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ABDULALI3468 — APPROVED** [review]
Hi @BezzB , 👍 

# APPROVED 👍 💯 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights 
✅App logic is working as expected.

## Optional suggestions

Well this format looks better than what you have used now:-

```
luxon.DateTime.local().toFormat("EEEE, LLLL d, y");
```

## Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear, and please remember to tag me in your question so I can receive the notification.

> * This is My **GitHub**: GitHub followers
> * This is My **Linkedin**: [Linkedin Badge](https://www.linkedin.com/in/abdul-ali-5400bb216/)&nbsp;
> * This is My **Twitter**: [Twitter Badge](https://twitter.com/mrabdul_ali)&nbsp;

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**